dataprofiler/tests/labelers/test_character_level_cnn_model.py

```diff
@@ -35,11 +35,12 @@
 }
 
 
-def mock_open(filename, *args):
+def mock_open(filename, *args, **kwargs):
     if filename.find("model_parameters") >= 0:
         return StringIO(json.dumps(mock_model_parameters))
     elif filename.find("label_mapping") >= 0:
         return StringIO(json.dumps(mock_label_mapping))
+    return StringIO()
```

Comment:
extra conditions to allow for debugging with IPDB. I think should stay for future cases if debugging is needed.